test/behave/steps/alarms.py

```diff
@@ -10,6 +10,7 @@
 @given('an alarm is set for {alarm_time}')
 def given_set_alarm(context, alarm_time):
     emit_utterance(context.bus, 'set an alarm for {}'.format(alarm_time))
+    time.sleep(3)
```

Comment:
`wait_for_dialog()` has an optional timeout arg, so you could set this to a higher value instead